embedded-elasticsearch/src/test/java/com/playtika/test/elasticsearch/springdata/SpringDataTest.java

```diff
@@ -27,9 +21,6 @@ public class SpringDataTest extends EmbeddedElasticSearchBootstrapConfigurationT
     @Autowired
     private ConfigurableListableBeanFactory beanFactory;
 
-    @Autowired
-    private NetworkTestOperations elasticsearchNetworkTestOperations;
```

Comment:
I had issues with the `PackageInstaller` after upgrading the docker image to elasticsearch 8.5.3. I think `yum` was not available in the new image. Since the test was disabled anyway, I decided to remove it along with the dependency to `NetworkTestOperations`.

Reply:
please redo it with toxiproxy as in other modules

Reply:
The support for `NetworkTestOperations` for this module was rather [undocumented](https://github.com/PlaytikaOSS/testcontainers-spring-boot/tree/develop/embedded-elasticsearch) apart from an minor implict hint in the documentation of `embedded.{module-name}.capabilities` in the [README in the project root](https://github.com/PlaytikaOSS/testcontainers-spring-boot/blob/develop/README.adoc). [The way ToxiproxyContainer is used](https://github.com/PlaytikaOSS/testcontainers-spring-boot/blob/develop/embedded-aerospike/src/main/java/com/playtika/test/aerospike/EmbeddedAerospikeBootstrapConfiguration.java#L54) in the other modules is also deprecated by now, so I am not sure whether it would make sense to also introduce this approach here.

I suggest to finish the upgrade to Spring Boot 3 first and to later add support for simulating network conditions for `embedded-elasticsearch` in a minor release of testcontainers-spring-boot 3.x.